src/main/java/se/sundsvall/precheck/service/utils/PreCheckUtil.java

```diff
@@ -0,0 +1,115 @@
+package se.sundsvall.precheck.service.utils;
+
+import generated.client.citizen.CitizenAddress;
+import generated.client.citizen.CitizenExtended;
+import generated.client.partyAssets.Asset;
+import jakarta.validation.constraints.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.zalando.problem.Problem;
+import se.sundsvall.precheck.api.model.PreCheckResponse;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+import static org.zalando.problem.Status.FORBIDDEN;
+
+public class PreCheckUtil {
+    public static final String CORRECT_ADDRESS_TYPE = "POPULATION_REGISTRATION_ADDRESS";
```

Comment:
Does this need to be public?

Reply:
I agree that the messages should not be publicly accessible in this class. 
Instead, I believe it to be more appropriate to move them to a separate class, possibly a 'Constants' class. This approach would help with the maintainability of the code somewhat, especially since these messages are also used in tests.

However, if this suggestion is also sub-optimal, please let me know!

Reply:
Let the variable have default visibility then?